### PR TITLE
Add buttons visual test

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
Addresses: https://github.com/blackbaud/skyux-theme/issues/40

### CODE REVIEW

**Comment by Blackbaud-SteveBrush** [comment]
@Blackbaud-AlexKingman This is ready for a look. It hooks up `@skyux/theme` to visual tests, and creates some screenshots for the button states.

**Comment by Blackbaud-AlexKingman** [comment]
Is the `buttons.e2e-spec.ts` file still needed? I noticed it also contains a bad link:
```
SkyHostBrowser.get('demos/button');
```

**Comment by Blackbaud-AlexKingman** [comment]
Also, is the `/visual/button` directory still needed? Notice there's a `buttons` and `button`.

File: e2e/buttons.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+describe('Buttons', () => {
+  function hoverMouseOverSelector(selector: string): any {
+    return browser.actions()
+      .mouseMove(element(by.css(`#screenshot-buttons-default-state ${selector}`)))
+      .perform();
+  }
+
+  beforeEach(() => {
+    SkyHostBrowser.get('visual/buttons');
+    SkyHostBrowser.setWindowBreakpoint('md');
```
Comment (Blackbaud-AlexKingman) [inline e2e/buttons.e2e-spec.ts]: I'm assuming we don't need extra breakpoints, because the buttons don't really have different states?

Comment (Blackbaud-SteveBrush) [reply e2e/buttons.e2e-spec.ts]: True. Here, I'm just setting the breakpoint to `md` for every test to make sure the screenshot's bounds are consistent.

**Comment by Blackbaud-SteveBrush** [comment]
@Blackbaud-AlexKingman This is ready for another look.

**Pull request merged**